app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -249,10 +256,27 @@ class WebViewActivity : BaseActivity(), io.homeassistant.companion.android.webvi
             // Allow showing this on the lock screen when using device controls panel
             setShowWhenLocked(intent.extras?.getBoolean(EXTRA_SHOW_WHEN_LOCKED) ?: false)
         }
-
+        enableEdgeToEdge()
         super.onCreate(savedInstanceState)
 
         binding = ActivityWebviewBinding.inflate(layoutInflater)
+
+        bottomInsetsApplied = serverManager.getServer()?.version?.isAtLeast(2026, 1) == true
+
+        ViewCompat.setOnApplyWindowInsetsListener(binding.root) { v, windowInsets ->
+            val insets = windowInsets.getInsets(WindowInsetsCompat.Type.systemBars())
+            val bottomInset = if (bottomInsetsApplied) 0 else insets.bottom
+
+            v.updatePadding(
+                top = insets.top,
+                bottom = bottomInset
+            )
+
+            rootInsets = Insets.of(insets.left, 0, insets.right, insets.bottom - bottomInset)
+
+            WindowInsetsCompat.CONSUMED
```

Comment:
Could make sense to call the default `onApplyWindowInsets` here so the `env` is set properly in case the bug is ever fixed in Android